examples/computer_administrators_script.py

```diff
@@ -0,0 +1,68 @@
+"""
+Script for display user groups and users in group Administrators
+
+Requirements: Python 2.7 or 3.4, pyWin32, pywinauto 0.5.3+
+ download the repo: https://github.com/pywinauto/pywinauto
+ place the script to the repo root folder
+"""
+from subprocess import Popen
+from pywinauto import application
+from pywinauto.timings import always_wait_until_passes
+
+
+class AccessDeniedError(StandardError):
+    """Raise when current user is not an administrator."""
+    def __init__(self, arg):
+        self.args = arg
+
+
+class NoExistGroupError(StandardError):
+    """Raise when group Administrators is not exist."""
+    def __init__(self, arg):
+        self.args = arg
+
+
+def is_user_an_admin():
+    """ Check user admin """
+    import os
+    if os.name == 'nt':
+        try:
+            # only windows users with admin privileges can read the C:\windows\temp
+            os.listdir(os.sep.join([os.environ.get('SystemRoot', 'C:\\windows'), 'temp']))
+        except WindowsError:
+            return False
+        else:
+            return True
+    else:
+        return 'SUDO_USER' in os.environ and os.geteuid() == 0
+
+
+if is_user_an_admin():
+    Popen(['compmgmt.msc'], shell=True)
+
+
+    @always_wait_until_passes(4, 2, application.ProcessNotFoundError)
+    def connect_to_mmc():
+        """ Returns connect app to instance """
+        return application.Application().connect(path="mmc.exe")
+
+
+    app = connect_to_mmc()
+    tree = app.mmc_main_frame.tree_view
+    groups = tree.get_item(r'\Computer Management (Local)\System Tools\Local Users and Groups\Groups')
+    groups.click()
+    items = app.mmc_main_frame.list_view.items()
+    print('Groups and descriptions:')

```

Comment:
For demo purpose it's better to output only group names (without description).
Also please start every new group of outputs with `\n`. For example, `print('\nGroups:')`.